docs/user_guide/selection/index.rst

```diff
@@ -151,9 +151,3 @@ Click below to find more details on how to use each one of the transformers.
 
 Additional Resources
 --------------------
-
```

Comment:
More details about feature selection can be found in the following resources:

- `Feature Selection Online Course <https://www.trainindata.com/p/feature-selection-for-machine-learning>`_
- `Feature Selection book <https://leanpub.com/feature-selection-in-machine-learning/>`_
- `Train in data's blog <https://www.blog.trainindata.com/>`_